lib/lambda_ethereum_consensus/fork_choice/helpers.ex

```diff
@@ -207,4 +212,58 @@ defmodule LambdaEthereumConsensus.ForkChoice.Helpers do
     current_epoch = Misc.compute_epoch_at_slot(current_slot)
     store.justified_checkpoint.epoch + 1 == current_epoch
   end
+
+  @spec root_by_id(atom() | Types.root() | Types.slot()) ::
+          {:ok, {Types.root(), boolean(), boolean()}} | {:error, String.t()} | atom()
+  def root_by_id(:head) do
+    with {:ok, current_status} <- BeaconChain.get_current_status_message(),
+         {:ok, signed_block} <- BlockStore.get_block(current_status.head_root) do
+      # TODO compute is_optimistic_or_invalid
+      execution_optimistic = true
+      {:ok, {signed_block.message.state_root, execution_optimistic, false}}
+    end
+  end
+
+  def root_by_id(:genesis), do: :invalid_id
```

Comment:
I think we should differentiate between "invalid" and "unsupported" values. Although, in this case we can just return `:not_found`